scalafmt-tests/src/test/resources/test/OperatorSpray.stat

```diff
@@ -90,9 +93,9 @@ object TestRoutes {
     pathPrefix("foo") {
       ???
     } ~
-    pathPrefix("bar") {
-      ???
-    }
+      pathPrefix("bar") {
```

Comment:
iirc, the original motivation for the "spray" indentation was to keep `pathPrefix` at the same indentation level. Do I understand correctly that the output introduced in this PR is the following

```scala
    pathPrefix("foo") {
      ???
    } ~
      pathPrefix("bar") {
        ???
      }
```

regardless whether `unindentTopLevelOperators` is true or false?

Reply:
Hmm, I'll review this more, let's hold off on merging then.  I might also be able to tackle the https://github.com/scalameta/scalafmt/issues/1370 issue as well.

Reply:
This isn't a code-related change from this PR, it simply indicates what happens in the current master branch.  This PR is "ready to merge".  This test was merged to help document/show the current behavior.  I'll create a separate issue/PR to figure out why that additional `~` is currently being indented on master.

Reply:
I think you were looking at the wrong part of the file, the formatted test shows it unindented in the expected: https://github.com/scalameta/scalafmt/blob/master/scalafmt-tests/src/test/resources/test/OperatorSpray.stat#L46-L51